build.sbt

```diff
@@ -36,10 +36,11 @@ inThisBuild(
     crossScalaVersions := List(scala213, scala212, scala211),
     resolvers += Resolver.sonatypeRepo("releases"),
     libraryDependencies ++= List(
-      scalatest.value % Test,
```

Comment:
wait a minute... are you switching to munit, from scalatest? can we support windows without that?

we tried it before, intellij doesn't run tests anymore.

Reply:
I switched to munit, because it was not possible to read the logs in the CI :/ 

Intellij should work now: https://blog.jetbrains.com/scala/2020/12/01/intellij-scala-plugin-2020-3/#munit-support

I can remove it and keep the changes for the Windows job only.